pkg/document/json/rga_tree_list.go

```diff
@@ -358,3 +492,20 @@ func (a *RGATreeList) insertAfter(
 
 	a.size++
 }
+
+func (a *RGATreeList) insertNodeAfter(
```

Comment:
It seems that insertNodeAfter has been newly added to solve the problem. Can you briefly describe the algorithm?

Reply:
Thank you for your comment!

Before `setByIndex` existed, only one element could be added when inserted into an array.
![image](https://user-images.githubusercontent.com/10924072/127732108-936ac068-3444-415b-a3bc-554ab48019e0.png)



So, when `setByIndex` is added and an operation such as movement occurs, the node of the index (with multiple elements) is moved, so the above method was added.
array 

![image](https://user-images.githubusercontent.com/10924072/127732118-5061fed3-02d6-49d3-81d0-6835c4f303a8.png)